### PR TITLE
Manga page redesign

### PR DESCRIPTION
**Pull request by NuckChorris**
So, this brings manga pages into the new design and new controller/route setup.

Which desperately needs refactor and deconfusilisation.

Only thing which needs fix is CSS on the cast.  Since there's no actors, the fade-on-mouseover needs to be nuked and we should probably center the name vertically

### CODE REVIEW

File: frontend/app/routes/manga/index.js [hunk 1]
```diff
@@ -1,12 +1,86 @@
...
+    loadFullCharacters: function() {
+      var manga_id = this.currentModel.get('id');
+      this.store.find('casting', { manga_id }).then((castings) => {
+        this.controller.set('fullCharacters', castings);
+      });
+    },
+
+    toggleFavorite: function() {
+      if (!this.get('currentUser.isSignedIn')) {
+        alert('Need to be signed in');
```
Comment (vevix) [inline frontend/app/routes/manga/index.js]: Is anime also like this? We should just not show the favorite button to guests.

File: frontend/app/templates/manga/index.hbs [hunk 2]
```diff
@@ -1,41 +1,131 @@
...
+    <div class="series-panel">
 
-<div class="series-description hidden-xs">
-  <p>{{truncate-text text=synopsis length=500}}</p>
-</div>
+      <div class="poster-container">
+        <div class="poster-actions">
+          <img class="series-poster" {{bind-attr src=posterImage}}>
+          {{#if libraryEntryExists}}
+            {{#if libraryEntry.isFavorite}}
```
Comment (vevix) [inline frontend/app/templates/manga/index.hbs]: Think this should be `mangaLibraryEntryExists` & `mangaLibraryEntry`

File: frontend/app/templates/manga/index.hbs [hunk 3]
```diff
@@ -1,41 +1,131 @@
...
+                <li><a {{action "setLibraryStatus" "Plan to Read"}}>Plan to Read</a></li>
+                <li><a {{action "setLibraryStatus" "Completed"}}>Completed</a></li>
+                <li><a {{action "setLibraryStatus" "On Hold"}}>On Hold</a></li>
+                <li><a {{action "setLibraryStatus" "Dropped"}}>Dropped</a></li>
+                <li><a {{action "removeFromLibrary"}}>Remove from Library</a></li>
+              </ul>
+            </div>
+            {{#if mangaLibraryEntryExists}}
+              <div class="personal-rating active">
+                {{awesome-rating type=currentUser.ratingType editable="true" action="setLibraryRating" rating=libraryEntry.rating}}
```
Comment (vevix) [inline frontend/app/templates/manga/index.hbs]: `mangaLibraryEntry`

File: frontend/app/routes/manga/index.js (hunk 1 above)
Comment (NuckChorris) [inline frontend/app/routes/manga/index.js]: We actually have it wrapped in `{{#if libraryEntryExists}}` so I'm not sure why the action checks again

File: frontend/app/templates/manga/index.hbs [hunk 4]
```diff
@@ -1,41 +1,131 @@
...
+            <small class="alternate-title">
+              {{model.englishTitle}}
+            </small>
+          {{/if}}
+        </h3>
+
+        <div class="series-infobar">{{partial "manga/infobar"}}</div>
+
+        <div class="synopsis">
+          <p>{{{truncate-text text=synopsis length=200 breakLines=true}}}</p>
```
Comment (vevix) [inline frontend/app/templates/manga/index.hbs]: `model.synopsis`

Comment (NuckChorris) [inline frontend/app/templates/manga/index.hbs]: Have I told you you're a meanie lately? Cause you are :sob:

File: frontend/app/templates/manga/index.hbs [hunk 5]
```diff
@@ -1,41 +1,131 @@
...
+            {{/each}}
+          </ul>
+        </div>
+
+        {{#if currentUser.isSignedIn}}
+          <div class="user-actions {{unless model.mangalibraryEntryExists 'no-entry'}}">
+            <div class="btn-group">
+
+              <button class="library-button library-entry dropdown-toggle {{if model.mangaLibraryStatus 'active'}}" type="button" data-toggle="dropdown">
+                {{#if mangalibraryEntryExists}}
```
Comment (vevix) [inline frontend/app/templates/manga/index.hbs]: `manga*L*ibraryEntryExists`

File: frontend/app/templates/manga/index.hbs [hunk 6]
```diff
@@ -1,41 +1,131 @@
...
+                <li><a {{action "setLibraryStatus" "Plan to Read"}}>Plan to Read</a></li>
+                <li><a {{action "setLibraryStatus" "Completed"}}>Completed</a></li>
+                <li><a {{action "setLibraryStatus" "On Hold"}}>On Hold</a></li>
+                <li><a {{action "setLibraryStatus" "Dropped"}}>Dropped</a></li>
+                <li><a {{action "removeFromLibrary"}}>Remove from Library</a></li>
+              </ul>
+            </div>
+            {{#if mangaLibraryEntryExists}}
+              <div class="personal-rating active">
+                {{awesome-rating type=currentUser.ratingType editable="true" action="setLibraryRating" rating=model.mangalibraryEntry.rating}}
```
Comment (vevix) [inline frontend/app/templates/manga/index.hbs]: manga*L*ibraryEntryExists

**Pull request merged**